pyramid_swagger/load_schema.py

```diff
@@ -46,45 +55,14 @@ def extract_query_param_schema(schema):
         return {
             'type': 'object',
             'properties': properties,
-            'additionalProperties': False,
-        }
-    else:
-        return None
-
-
-def extract_path_schema(schema):
-    """Extract a schema for path variables for an endpoint.
-
-    As an example, this would take this swagger schema:
-        {
-            "paramType": "path",
-            "type": "string",
-            "enum": ["foo", "bar"],
-            "required": true
-        }
-    To this jsonschema:
-        {
-            "type": "string",
-            "enum": ["foo", "bar"],
-        }
-    Which we can then validate against a JSON object we construct from the
-    pyramid request.
-    """
-    properties = dict(
-        (s['name'], strip_swagger_markup(s))
-        for s in schema['parameters']
-        if s['paramType'] == 'path'
-    )
-    if properties:
-        return {
-            'type': 'object',
-            'properties': properties,
-            'additionalProperties': False,
+            # Allow extra headers
+            'additionalProperties': param_type == 'header',
```

Comment:
Ah, because there are often headers outside of the spec for "normal http" requests. Host, User-Agent, Accepts, etc. 

I'll add more detail to this comment.